checks.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
How about leaving a comment at the top briefly describing what this file does and how it's expected to be used?